examples/PathHandler.js

```diff
@@ -76,9 +77,10 @@ PathHandler.prototype.arcAbs = function(rx, ry, xAxisRotation, largeArcFlag, swe
  *  @param {Number} x
  *  @param {Number} y
  */
-PathHandler.prototype.arcRel = function(rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y) {
+PathHandler.prototype.arcRel = function(/* rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y */) {
```

Comment:
Same comment regarding commenting out the function parameters

Reply:
Ok, done in the manner mentioned above. Let me know if you want it done in that manner (through `eslint-disable-next-line`).

Reply:
LGTM